packages/medusa-payment-stripe/package.json

```diff
@@ -16,19 +16,23 @@
   "scripts": {
     "prepare": "cross-env NODE_ENV=production yarn run build",
     "test": "jest --passWithNoTests src",
-    "build": "tsc",
+    "build": "tsc -p ./tsconfig.server.json && medusa-admin bundle",
     "watch": "tsc --watch"
   },
   "devDependencies": {
+    "@medusajs/admin": "^6.0.0",
     "@medusajs/medusa": "^1.12.0",
     "@types/stripe": "^8.0.417",
     "awilix": "^8.0.1",
     "cross-env": "^5.2.1",
     "jest": "^25.5.4",
+    "medusa-react": "^9.0.0",
     "typescript": "^4.9.5"
   },
   "peerDependencies": {
-    "@medusajs/medusa": "^1.12.0"
+    "@medusajs/admin": "^6.0.0",
+    "@medusajs/medusa": "^1.12.0",
+    "medusa-react": "^9.0.0"
```

Comment:
Suggestion : Shouldn't this be an optional peers in case a user doesn't want to use the widget? Cc @kasperkristensen

Reply:
`@medusajs/admin` can just be a `devDependency` as it is only used building the plugin. As for `medusa-react` then yes it could be optional for the people that don't use `@medujsajs/admin` at all :)

Reply:
Yeah so medusa react and admin should be peer optional and dev deps 👌